rskj-core/src/test/java/co/rsk/peg/BridgeSupportAddSignatureTest.java

```diff
@@ -62,6 +62,7 @@ class BridgeSupportAddSignatureTest {
 
     private final BridgeConstants bridgeRegTestConstants = BridgeRegTestConstants.getInstance();
     private final NetworkParameters btcRegTestParams = bridgeRegTestConstants.getBtcParams();
+    private WhitelistSupport whitelistSupport;
```

Comment:
nit: move this variable [here](https://github.com/rsksmart/rskj/blob/bugfix/issues-whitelist-refactor/rskj-core/src/test/java/co/rsk/peg/BridgeSupportAddSignatureTest.java#L69-#L71)